blazor/datagrid/columns.md

```diff
@@ -511,6 +511,74 @@ In the following code sample, you can prevent default filter query generation us
 
 > You can find the fully working sample [here](https://github.com/SyncfusionExamples/blazor-datagrid-prevent-query-generation-for-foriegnkey-column).
 
+### Customize filter UI in foreignkey column
+
+A custom component can be rendered in the filter UI of the foreign key column using the [FilterTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridColumn.html#Syncfusion_Blazor_Grids_GridColumn_FilterTemplate) feature of Grid.
+
+The following example demonstrates the way to create a custom filtering UI by rendering the `SfDropDownList` component in the foreign key column.
+
+> * For all filter types other than FilterBar, filtering parameters will be sent in the form of `PredicateModel<T>`.
```

Comment:
for notes it is single point only the second point is continuation for first point and move the notes before the following sample topic